site/static/main.js

```diff
@@ -1,36 +1,34 @@
 /* tslint:disable:no-unused-variable */
```

Comment:
why?  I think it's pretty annoying and didn't like this to be TS from the beginning.  This is definitely an overkill.